kbase-extension/scss/modules/_mixins.scss

```diff
@@ -34,3 +34,8 @@
 @mixin default-container-padding {
   padding: $padding-large-horizontal;
 }
+
+@mixin advanced-hidden {
+  font-style: italic;
+  margin-left: 6px;
+}
```

Comment:
this is used for all the advanced/hidden params, so turn it into a mixing and remove all the inline css that specifies these styles